app/src/controllers/object.js

```diff
@@ -202,6 +202,14 @@ const controller = {
       const objects = [];
       const userId = await userService.getCurrentUserId(getCurrentIdentity(req.currentUser, SYSTEM_USER));
 
+      let bucketId = undefined;
+      bb.on('field', (fieldname, val) => {
```

Comment:
just wondering if the bucketId is going to be a query parameter (eg: POST `/object?bucketId=oiuo-r3434r34-3d4rd4-3445rtf-3ewedwe`) can't you get it using `req.query.bucketId` instead of in the busboy field event ? i think we're only uploading to a single bucket.

Reply:
That could be an option yeah. Worth discussing with @jujaga as he's moved this into his other PR too

Reply:
Slipstreamed into #83